src/main/resources/edu/cornell/kfs/fp/businessobject/datadictionary/IncomingWireAchMapping.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xmlns:dd="http://rice.kuali.org/dd" xsi:schemaLocation=" http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd http://rice.kuali.org/dd http://rice.kuali.org/dd/dd.xsd">
+
+<bean id="IncomingWireAchMapping" parent="IncomingWireAchMapping-parentBean"/>
+
+	<bean id="IncomingWireAchMapping-parentBean" abstract="true" parent="BusinessObjectEntry">
+		<property name="businessObjectClass" value="edu.cornell.kfs.fp.businessobject.IncomingWireAchMapping"/>
+		<property name="objectLabel" value="IncomingWireAchMapping"/>
+		<property name="titleAttribute" value="IncomingWireAchMapping"/>
+		<property name="inquiryDefinition">
+			<ref bean="IncomingWireAchMapping-inquiryDefinition"/>
+		</property>
+		<property name="lookupDefinition">
+			<ref bean="IncomingWireAchMapping-lookupDefinition"/>
+		</property>
+		<property name="attributes">
+			<list>
+				<ref bean="IncomingWireAchMapping-shortDescription"/>
+				<ref bean="IncomingWireAchMapping-longDescription"/>
+				<ref bean="IncomingWireAchMapping-notes"/>
+				<ref bean="IncomingWireAchMapping-chartOfAccountsCode"/>
+				<ref bean="IncomingWireAchMapping-accountNumber"/>
+				<ref bean="IncomingWireAchMapping-subAccountNumber"/>
+				<ref bean="IncomingWireAchMapping-financialObjectCode"/>
+				<ref bean="IncomingWireAchMapping-financialSubObjectCode"/>
+				<ref bean="IncomingWireAchMapping-projectCode"/>
+				<ref bean="IncomingWireAchMapping-organizationReferenceId"/>
+				<ref bean="IncomingWireAchMapping-active"/>
+			</list>
+		</property>
+	</bean>  
+    
+            
+    <bean id="IncomingWireAchMapping-shortDescription" parent="IncomingWireAchMapping-shortDescription-parentBean"/>
+    <bean id="IncomingWireAchMapping-shortDescription-parentBean" abstract="true" parent="AttributeDefinition">
+    	<property name="name" value="shortDescription"/>
+    	<property name="label" value="Short Description"/>
+    	<property name="shortLabel" value="Short Desc"/>
+    	<property name="maxLength" value="30"/>
+    	<property name="forceUppercase" value="true"/>
+        <property name="required" value="true"/>
+    	<property name="control">
+            <bean parent="TextControlDefinition" p:size="35"/>
+        </property>
+    	<property name="validationPattern">
+      		<bean parent="AnyCharacterValidationPattern">
```

Comment:
Any particular reason why this file needs to make beans extending from "AnyCharacterValidationPattern" directly? KFS should already have a child bean of "AnyCharacterValidationPattern" that is configured to allow for whitespace, and the interior max-length check seems redundant since the attribute definition already defines one.